README.md

```diff
@@ -1,7 +1,9 @@
 # ui
 Published assets for Cockroach Labs user interfaces
 
-Commands for local setup:
-
-`npm run init` - for initial run, will install top level dependencies, package level ones, run lerna link
-`npm start` - run in root of the project to run dev servers for all sub packages.<br/>
\ No newline at end of file
+## Development setup
```

Comment:
please make corresponding changes in packake.json as init command seems to be outdated

Reply:
Done